### PR TITLE
Update wording for “Authenticating” reference

### PR DESCRIPTION
**Pull request by EmilyM1**
This is a fix for issue 21955 for CSR instructions.
Issue here: https://github.com/kubernetes/website/issues/21955

### CODE REVIEW

File: content/en/docs/reference/access-authn-authz/certificate-signing-requests.md [hunk 1]
```diff
@@ -284,7 +283,7 @@ Retrieve the Certificate from the CSR.
 kubectl get csr/john -o yaml
 ```
 
-The Certifcate value is in Base64 format, under status.certificate.
+The Certificate value is in encrypted with Base64 format under status.certificate.
```
Comment (liggitt) [inline content/en/docs/reference/access-authn-authz/certificate-signing-requests.md:286]:
```suggestion
The Certificate value is in Base64-encoded format under status.certificate.
```

File: content/en/docs/reference/access-authn-authz/certificate-signing-requests.md [hunk 2]
```diff
@@ -241,7 +240,7 @@ openssl req -new -key john.key -out john.csr
 
 ### Create Certificate Request Kubernetes Object
 
-Create a CertificateSigningRequest and submit it to Kubernetes Cluster via kubectl. Below is script to generate one.
+Create a CertificateSigningRequest and submit it to a Kubernetes Cluster via kubectl. Below is a script to generate the certificate.
```
Comment (liggitt) [inline content/en/docs/reference/access-authn-authz/certificate-signing-requests.md:243]:
```suggestion
Create a CertificateSigningRequest and submit it to a Kubernetes Cluster via kubectl. Below is a script to generate the CertificateSigningRequest.
```

**Comment by EmilyM1** [comment]
Solid suggestions @liggitt
Fixed in commit 2c6ba2c

File: content/en/docs/reference/access-authn-authz/certificate-signing-requests.md [hunk 3]
```diff
@@ -228,7 +227,7 @@ rules:
 
 ## Normal User
 
-Few steps are required in order to get normal user to be able to authenticate and invoke API. First, this user must have certificate issued by the Kubernetes Cluster, and then present that Certificate into the API call as the Certificate Header, or through the kubectl.
+There are a few steps are required in order to get normal user to be able to authenticate and invoke API. First, this user must have certificate issued by the Kubernetes Cluster, and then present that Certificate into the API call as the Certificate Header, or through the kubectl.
```
Comment (kbhawkey) [inline content/en/docs/reference/access-authn-authz/certificate-signing-requests.md:230]: @EmilyM1 , These sentences could be reworded. Something like:

To set up a user for authentication to the API server, follow these steps:
1. Obtain a certificate issued by the Kubernetes cluster
1. Add the issued certificate as the certificate header to the API request or apply the certificate with `kubectl`

**Pull request merged**